apps/charterafrica/src/lib/data/common/index.js

```diff
@@ -121,13 +129,15 @@ export async function processPageFellowships({ blocks }) {
         updatedAt: "2023-01-26T11:21:14.868Z",
         url: "/images/charter-africa-brand.svg",
       },
-      deadline: "2023-02-11",
+      link: {
+        href: `/grants/${i}`,
```

Comment:
?

Reply:
@kilemensi Not sure what the issue is here